experimental/dds/tree2/src/feature-libraries/schemaBuilderBase.ts

```diff
@@ -140,11 +140,62 @@ export class SchemaBuilderBase<
 		});
 	}
 
+	/**
+	 * Finalizes the schema builder, producing {@link SchemaLibrary}s that capture the content added to this builder
+	 * and any additional `SchemaLibrary`s that were added to it.
+	 *
+	 * @remarks
+	 * May only be called once after adding content to builder is complete.
+	 *
+	 * @returns A {@link SchemaLibrary} usable with future `SchemaBuilder`s
+	 */
+	public finalize(): SchemaLibrary;
+
+	/**
+	 * Finalizes the schema builder, producing SchemaLibraries that capture the content added to this builder,
+	 * any additional SchemaLibraries that were added to it, and the schema of a root field.
+	 *
+	 * @remarks
+	 * May only be called once after adding content to builder is complete.
+	 *
+	 * @param root - The root field schema.
+	 * @typeParam TSchema - The kind of field schema of `root`.
+	 *
+	 * @returns A {@link TypedSchemaCollection} that may be used in the construction of trees.
+	 */
+	public finalize<const TSchema extends ImplicitFieldSchema>(
+		root: TSchema,
+	): TypedSchemaCollection<NormalizeField<TSchema, TDefaultKind>>;
+
+	/**
```

Comment:
FYI you don't need this doc on the implementation; I don't think it will ever show up in intellisense or mouse hover or anything.

Reply:
Removed